ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -188,5 +188,7 @@ conformance_test(
         "--verbose",
         "--concurrent-test-runs=1",  # lowered from default #procs to reduce flakes - details in https://github.com/digital-asset/daml/issues/7316
         "--include=ParticipantPruningIT",
+        # Disable tests targeting only multi-participant setups
+        "--exclude=ParticipantPruningIT:PRDisclosureAndRetroactiveDivulgence",
```

Comment:
Canton can support multi-participant setups, and the port list above let's me believe it does so here. What fails if you enable the test?

Reply:
I was not sure regarding this one so I've disabled it for peace-of-mind knowing that I'll retouch these with the PR for adapting conformance tests for Canton. I'll remove, now that you have mentioned it.